docs/reference/search/field-caps.asciidoc

```diff
@@ -34,6 +34,10 @@ GET /_field_caps?fields=rating
 The field capabilities API returns the information about the capabilities of
 fields among multiple indices.
 
+Use the field capabilities API to return <<runtime,runtime fields>> like any
+other field. For example, a runtime field with a `runtime_type` of
+`keyword` returns as any other field that belongs to the `keyword` family.
```

Comment:
runtime_type no longer exists

Reply:
I'll change `runtime_type` to just "type".